### PR TITLE
Sync moban with coala-community repo

### PR DESCRIPTION
**Pull request by KVGarg**
The Travis CI moban build is failing
due to outdated files.

Closes https://github.com/coala/community/issues/241

### CODE REVIEW

**Comment by sks444** [comment]
Add this commit to https://github.com/coala/community/pull/244 with another commit, so that, that pr passes. Both are dependent to each other only one could be merged.

**Comment by jayvdb** [comment]
My comments on #244 also related to this PR.  Thanks @sks444 and @KVGarg for ensuring these two PRs stacked nicely so they can be merged together when we have the desired solution ;-).

File: setup.cfg [hunk 1]
```diff
@@ -2,7 +2,7 @@
 python-tag = py34.py35.py36
 
 [tool:pytest]
-minversion = 3.6.1
+minversion = 4.4.0
```
Comment (jayvdb) [inline setup.cfg]: this should be now 3.6.4 as that is what you believe will work

File: test-requirements.txt [hunk 2]
```diff
@@ -12,7 +12,7 @@ coverage-config-reload-plugin~=0.2
 codecov~=2.0.5
 moban~=0.3.3 ; python_version > '3.0'
 packaging~=16.8
-pytest~=3.6.1
+pytest~=4.4.0
```
Comment (jayvdb) [inline test-requirements.txt]: this should be now 3.6.4 as that is what you believe will work

**Pull request merged**